### ISSUE TITLE
Missing icons

### ISSUE THREAD

**Issue by japonecka** [sixsixfive/Hedera#8]
Hello, after today's update from arch I am missing Hedera icons. Currently icons (whole icon folder) are missing even here. Is that a mistake or intention? These icons were stunning and I loved using these.

**Comment by sixsixfive** [comment]
yeah sorry the icons are now here: https://github.com/sixsixfive/Hedera-icons
